app/views/collection_drafts/show.html.erb

```diff
@@ -46,7 +46,7 @@
           <% end %>
         <% end %>
 
-        <%= link_to "Delete #{name_to_title(resource_name)}", "#delete-draft-modal", class: 'display-modal' %>
+        <%= link_to "Delete #{name_to_title(resource_name)}", "#delete-draft-modal", class: 'display-modal eui-btn--link' %>
```

Comment:
As a consequence of the eui change, I noticed that this link was slightly off.  This fixes it.